src/main.rs

```diff
@@ -213,29 +212,45 @@ async fn reload_network(auth: BearerAuth) -> impl Responder {
     }
 }
 
-async fn post(path: &str, auth: Option<BearerAuth>) -> Result<HttpResponse> {
-    if let Some(auth) = auth {
-        if !verify_token(auth)? {
-            error!("post {path} verify false");
-            return Ok(HttpResponse::build(StatusCode::UNAUTHORIZED).finish());
-        }
-    }
-
-    let mut sender = match sender().await {
+async fn post_with_json_body(path: &str, body: Option<impl Serialize>) -> Result<HttpResponse> {
```

Comment:
nit: as we distinguish `post_with_json_body` from `post_with_empty_body` now, can't `body` here simply be: `impl Serialize` (without Option)?

Reply:
Done ;)